onboard/nav/search/searchStateMachine.hpp

```diff
@@ -1,27 +1,30 @@
-#ifndef SEARCHER_HPP
-#define SEARCHER_HPP
+#ifndef SEARCH_STATE_MACHINE_HPP
+#define SEARCH_STATE_MACHINE_HPP
 
 #include "rover.hpp"
 #include "utilities.hpp"
 
 class StateMachine;
 
-class Searcher {
+enum class SearchType
+{
+    SPIRALOUT,
+    LAWNMOWER,
+    SPIRALIN,
+    UNKNOWN
```

Comment:
This is the same as for Avoidance type, why do we need `Unkown`? Where you use it in the code, you should just do `default` rather than `case: Unknown`